src/angular-local-storage.js

```diff
@@ -302,7 +302,10 @@ angularLocalStorage.provider('localStorageService', function() {
           expiryDate.setTime(expiryDate.getTime() + (-1 * 24 * 60 * 60 * 1000));
           expiry = "; expires=" + expiryDate.toGMTString();
           value = '';
-        } else if (isNumber(daysToExpiry) && daysToExpiry !== 0) {
+          //if the value is undefined set the cookie to session cookie
+        } else if (daysToExpiry == undefined) {
```

Comment:
this is the solution in plain javascript:
document.cookie = "cookiename=value; expires=0; path=/";
so there should be a way to set the expires in a date or also session